workers/ohsome_quality_analyst/api.py

```diff
@@ -211,6 +217,42 @@ async def _fetch_report(
     return response
 
 
-@app.get("/regions")
+@app.get("/list_regions")
 async def get_available_regions():
+    """List names of available regions."""
     return await db_client.get_available_regions()
+
+
+@app.get("/list_indicators")
+async def list_indicators():
+    """List names of available indicators."""
+    return list(load_metadata("indicators").keys())
```

Comment:
The return type should be `JSON` (or `dict` in python) as it is defined in the openapi.json.

See:
```bash
curl -s http://localhost:8080/openapi.json | python3 -m json.tool | grep "\/list_indicators" -A16
        "/list_indicators": {
            "get": {
                "summary": "List Indicators",
                "description": "List names of available indicators.",
                "operationId": "list_indicators_list_indicators_get",
                "responses": {
                    "200": {
                        "description": "Successful Response",
                        "content": {
                            "application/json": {
                                "schema": {}
                            }
                        }
                    }
                }
            }
        },
```
I would suggest something like this (@matthiasschaub please check if this response fits into the other API endpoints):
```suggestion
    response = empty_api_response()
    response["indicator_names"] = list(load_metadata("indicators").keys())
    return response
```

The same applies to the other new endpoints.

Reply:
I think you are right. The response should be a JSON (Python dict). Using the `emtpy_api_response` function for a template is a good idea, but the `attribution` field of the template is not needed and also not correct. Maybe writting two template functions would make sense.

```python
def api_resonse_base_template(request_url: str) -> dict:
    return {"apiVersion": oqt_version}


def api_response_template(request_url: Optional[str] = None) -> dict:
    template = api_resonse_base_template()
    if request_url is not None:
        template["requestUrl"] = request_url
    template["attribution"] = {
        "text": "© OpenStreetMap contributors",
        "url": "https://ohsome.org/copyrights",
    }
    return template.copy()

@app.get("/list_indicators")
async def list_indicators():
    """List names of available indicators."""
    response = api_resonse_base_template()
    response["indicator_names"] = list(load_metadata("indicators").keys())
    return response
```

Reply:
At first I thought we could simply return a list, which is also a valid JSON response (JSON Array). But there are some reasons why this considered by some as no good practice. https://softwareengineering.stackexchange.com/questions/286293/whats-the-best-way-to-return-an-array-as-a-response-in-a-restful-api